nxdrive/utils.py

```diff
@@ -1286,3 +1289,47 @@ def compute_digest(path: Path, digest_func: str, callback: Callable = None) -> s
         return UNACCESSIBLE_HASH
 
     return str(h.hexdigest())
+
+
+def config_paths() -> Tuple[List[Path], Path]:
+    """Return the list of possible local configuration paths and the default one."""
+    conf_name = "config.ini"
+    paths = (
+        Path(sys.executable).parent / conf_name,
+        Path(Options.nxdrive_home) / conf_name,
+        Path(conf_name),
+    )
+    return list(paths), paths[1]
+
+
+def save_config(section: str, config_dump: Dict[str, Any]) -> Path:
```

Comment:
You could also simplify by removing the `section` arg. It is not meant to be changed.